tests/AgentTests/NotificationTests.hs

```diff
@@ -268,26 +268,39 @@ testNotificationSubscriptionNewConnection APNSMockServer {apnsQ} = do
     (bobId, qInfo) <- createConnection alice SCMInvitation
     liftIO $ threadDelay 500000
     aliceId <- joinConnection bob qInfo "bob's connInfo"
+    liftIO $ print 1
```

Comment:
just as a reminder to remove prints before merging

Reply:
I am debugging the failing test